modules/katello_sync_plan.py

```diff
@@ -50,13 +51,16 @@
     description:
       - Name of the Katello sync plan
     required: true
+  description:
+    description:
+      - Description of the Katello sync plan
   organization:
     description:
       - Organization that the sync plan is in
     required: true
   interval:
     description:
-      - How often synchronization should run ('hourly', 'daily', 'weekly')
+      - How often synchronization should run ('hourly', 'daily', 'weekly', 'custom cron')
     required: true
```

Comment:
Can you add the `choices` key? https://github.com/theforeman/foreman-ansible-modules/blob/4b4b927443f3be50dbbf8f93d94ba74b2e0b295b/modules/katello_repository.py#L68-L74